frontend/packages/kubevirt-plugin/integration-tests/tests/utils/types.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-undef */
+
+export type provisionOption = {
```

Comment:
Please use CamelCase when declaring types, i.e. `ProvisionOption`.

(Also in other files, if relevant.)

Reply:
Fixed.